documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue

```diff
@@ -238,6 +241,7 @@ export default {
     openPreview() {
       this.loading = true;
       if (this.file && this.file.folder){
+        this.file.name = decodeURI(this.file.name);
```

Comment:
we should avoid changing retrieved information and keep it as it is. BTW, you can add a dedicated data variable with different name such as `fileName` and assing into it the decoded value.

Reply:
i leave this change i find is not necessary after a test of fixed behavior